ui-kit/Button/Button.styles.js

```diff
@@ -136,7 +136,9 @@ const Button = styled.button`
   font-family: ${themeGet('fonts.base')};
   font-size: ${themeGet('fontSizes.base')};
   font-weight: ${themeGet('fontWeights.bold')};
-  line-height: 2.5;
+  line-height: 1.5;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```

Comment:
@codingwithlis please update this to use Theme Values instead. Likely to be `s` or `xs`

Reply:
Done👍